tasks/test.py

```diff
@@ -8,14 +8,26 @@
 from invoke import task
 from invoke.exceptions import Exit
 
-from .constants import AGENT_BASED_INTEGRATIONS, ROOT
+from .constants import AGENT_BASED_INTEGRATIONS, ROOT, FILE_EXTENSIONS_TO_TEST
+
+
+def files_requiring_tests(files):
+    """
+    Given a list of files, return the files that have an extension listed in FILE_EXTENSIONS_TO_TEST
+    """
+    files_to_test = []
+    for file in files:
+        if file and file.endswith(tuple(FILE_EXTENSIONS_TO_TEST)):
+            files_to_test.append(file)
+
+    return files_to_test
```

Comment:
Not a fan of coding golf but this could actually be
```python
return list(f for f in files if f and f.endswith(tuple(FILE_EXTENSIONS_TO_TEST)))
```